arbitrum-docs/launch-orbit-chain/orbit-sdk-introduction.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Introduction to the Arbitrum Orbit SDK'
+sidebar_label: 'Orbit SDK: Introduction'
+description: 'introduction to the Arbitrum Orbit SDK'
+author: Mehdi Salehi
+sme: Mehdi Salehi
+target_audience: 'Developers deploying and maintaining Orbit chains.'
+sidebar_position: 0
+---
+
+import PublicPreviewBannerPartial from './partials/_orbit-public-preview-banner-partial.md';
+
+<PublicPreviewBannerPartial />
+
+
+The Arbitrum Orbit SDK is a comprehensive toolkit designed for intermediate blockchain developers interested in building on the Arbitrum Orbit platform.
+
+This SDK facilitates the creation and management of your own Orbit chain using the advanced capabilities of Viem, a modern alternative to traditional Ethereum libraries like _ethers.js_ and _web3.js_.
+
+## Arbitrum Orbit Chains
+
+The Arbitrum Orbit SDK helps you to spin up an <a data-quicklook-from="arbitrum-orbit">Orbit</a> chain as a <a data-quicklook-from="layer-3-l3">Layer 3 (L3)</a> chain settling transactions with any <a data-quicklook-from="layer-2-l2">Layer 2 (L2)</a> chains, or as a Layer 2 (L2) chain directly settling on <a data-quicklook-from="layer-1-l1">Layer 1 (L1)</a> chains. This adaptability opens up a realm of possibilities for application development and blockchain use cases.
+
+### Enhanced Development Experience
```

Comment:
This is more on Product's side, but I would simplify the introduction by removing this subsection, since it doesn't add a lot of value to the reader (IMO)

Reply:
Merged manually.